vkcs/internal/services/monitoring/requests.go

```diff
@@ -35,11 +36,14 @@ func CreateUser(client *gophercloud.ServiceClient, projectID string, opts Create
 }
 
 func DeleteUser(client *gophercloud.ServiceClient, projectID string, userID string) (r gophercloud.Result) {
+	resp, err := client.Delete(fmt.Sprintf("%s/%s", serviceUserURL(client, projectID), userID), &gophercloud.RequestOpts{
+		MoreHeaders: map[string]string{
+			"Content-Type": "application/json",
+		},
+		OkCodes: []int{http.StatusOK, http.StatusAccepted, http.StatusNotFound},
```

Comment:
404 should not be OK, you should handle it in resource's delete method.

Reply:
By the way, you should only accept the code that API returns, does it really returns 202?

Reply:
Moved the 404 processing to the monitoring resource code. Delete User really returns 202 code